project_event/i18n/fr.po

```diff
@@ -687,7 +686,7 @@ msgstr "Horaire Prévu"
 #: model:ir.ui.view,arch_db:project_event.res_partner_sector_for_view
 #: model:ir.ui.view,arch_db:project_event.view_project_project_search
 msgid "Faculty Sectors"
-msgstr "Secteur de Faculté"
+msgstr "Secteur"
```

Comment:
Not the corresponding translation of the english text

Reply:
It correspond to the functional spec